MovieApp/MovieApp/App/Features/Movies/ViewControllers/FavoriteMoviesViewController/FavoriteMoviesPresenter.swift

```diff
@@ -1,10 +1,19 @@
 import UIKit
+import Combine
 
 class FavoriteMoviesPresenter {
     
     private let moviesUseCase: MoviesUseCaseProtocol
     
     var data = [MovieViewModel]()
```

Comment:
no need foir this

Reply:
Without combine imported, Xcode can't recognise AnyPublisher on line 10

Reply:
I still need it in this branch because I didn't implement diffableDataSource. It is removed in next PR